habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -59,10 +59,17 @@ def _setup_actor_critic_agent(self, ppo_cfg: Config) -> None:
         """
         logger.add_filehandler(self.config.LOG_FILE)
 
-        self.actor_critic = PointNavBaselinePolicy(
+        policy = baseline_registry.get_policy(self.config.RL.POLICY.name)
+        self.actor_critic = policy(
             observation_space=self.envs.observation_spaces[0],
             action_space=self.envs.action_spaces[0],
             hidden_size=ppo_cfg.hidden_size,
+            rnn_type=self.config.RL.DDPPO.rnn_type,
```

Comment:
I had a second look on this design choice, accessing `RL.DDPPO` looks strange here, as it's different baseline. Let's remove new unrelated arguments and let's the run fails if wrong Policy was specified.

Reply:
Done!